src/main/java/org/mifos/processor/bulk/camel/routes/BatchStatusRoute.java

```diff
@@ -52,7 +52,7 @@ public void configure() throws Exception {
                 .unmarshal().json(JsonLibrary.Jackson, BatchDTO.class).process(exchange -> {
                     BatchDTO batchSummary = exchange.getIn().getBody(BatchDTO.class);
 
-                    long percentage = (long) (((double) batchSummary.getSuccessful() / batchSummary.getTotal()) * 100);
+                    int percentage = (int) (((double) batchSummary.getSuccessful() / batchSummary.getTotal()) * 100);
```

Comment:
@fynmanoj  We encountered this issue because while fetching COMPLETION_RATE from exchange, it fetched null and then it called `intValue()` on null. 
NPE wasn't happening at this line. It occurred in other file (BatchStatusWoker). 

Here I am making the data type to int. This is due to 2 reasons
1. percentage value got from 0-100 so no need to have it long, its an overkill.
2. In the worker, we are initializing this percentage value as int so I do not think we should store it as long in the camel route's and then fetch it as Long in worker and then convert it to int using method `intValue()`

Reply:
nvm I understood. Thanks